packages/react/src/constants/CardPropTypes.js

```diff
@@ -160,7 +160,7 @@ export const TableCardPropTypes = {
           href: PropTypes.string,
           target: PropTypes.string,
         }),
-        /** optional type. If "TIMESTAMP", it will format to 'L HH:mm' */
+        /** optional type. If "TIMESTAMP", it will format to 'L HH:mm:ss' */
         type: PropTypes.string,
       })
```

Comment:
The `defaultDateFormatPattern` prop us not added to the `CardPropTypes` for BarChartCard, ComboChartCard. TableCard and TimeSeriesCard?